lib/cocaine/command_line/runners/posix_runner.rb

```diff
@@ -4,10 +4,15 @@ module Cocaine
   class CommandLine
     class PosixRunner
       def self.available?
-        require 'posix/spawn'
-        true
-      rescue LoadError
-        false
+        return @available unless @available.nil?
+
+        @available ||=
+          begin
+            require 'posix/spawn'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.